app/views/devise/sessions/new.html.haml

```diff
@@ -0,0 +1,17 @@
+%h2 Log in
```

Comment:
if you haven't modified this please remove it

Reply:
also out of scope

Reply:
Removed.